extension/js/common/oauth2/oauth2_inject.ts

```diff
@@ -0,0 +1,10 @@
+/* ©️ 2016 - present FlowCrypt a.s. Limitations apply. Contact human@flowcrypt.com */
+
+// Decleared win variable to avoid `Type 'string' is not assignable to type 'Location | (string & Location)'.ts(2322)` error
+const win: Window = window;
+
+// Redirect back to the extension itself so that we have priveledged
+// access again
```

Comment:
Please update this comment to explain what type of privileged access this is, and what was it needed for that can't be done directly from here.